sdk/eventhub/mock-hub/samples/javascript/src/ehSample.js

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+const { readFileSync } = require('fs');
+const { resolve: resolvePath } = require('path');
+const { MockEventHub } = require("@azure/mock-hub");
```

Comment:
Are we thinking of publishing this package? Or is it just assumed they'll npm pack it themselves?

Reply:
Honestly right now the samples are just for our own reference, since I don't know if/when we plan to publish the package.

Reply:
(I figured, but just wanted to check)